picard/ui/itemviews.py

```diff
@@ -144,7 +144,7 @@ def __init__(self, window, parent=None):
         }
         FileItem.file_colors = {
             File.NORMAL: TreeItem.text_color,
-            File.CHANGED: config.setting["color_modified"],
+            File.CHANGED: TreeItem.text_color,
```

Comment:
Do i miss something or this is setting the same color for Normal and Changed files ?

Reply:
It is, that's another reason why I removed that option. The reason probably is that we render that red background color for changed files anyway. Here is a screenshot how it looks like in Picard:

![grafik](https://user-images.githubusercontent.com/29852/48426749-ade65000-e767-11e8-9780-5427c589b02a.png)